PLANNING.md

```diff
@@ -0,0 +1,70 @@
+# Mobile Team Releases Planning
+
+This document describes how the mobile team is planning each one of the releases.
+
+**What’s a release?**
+
+The release is every version of the app that’s sent to the store as a production release. The release is always a major/minor update, like from (1.0.0 to 1.1.0 or to 2.0.0). Patch releases (from 1.0.0 to 1.0.1) won’t follow this steps and are considered hotfixes releases. Our versioning is following the [Semantic Versioning 2.0.0](https://semver.org) guide.
```

Comment:
follow **these** steps instead of follow **this** steps in 2nd line, imo.

Reply:
👍